xbmc/pvr/providers/PVRProviders.h

```diff
@@ -43,6 +43,12 @@ class CPVRProvidersContainer
      */
   std::vector<std::shared_ptr<CPVRProvider>> GetProvidersList() const;
 
+  /*!
+     * Get the number of providers in this container
+     * @return The total number of providers
+     */
+  int GetNumProviders() const;
```

Comment:
Signed ? Can there be less than 0 providers? ;-)

Reply:
Whoops, fixed :)